packages/flutter/lib/src/services/text_input.dart

```diff
@@ -953,10 +953,15 @@ mixin TextSelectionDelegate {
 
 /// An interface to receive information from [TextInput].
 ///
+/// If [TextInputConfiguration.enableDeltaModel] is set to true you should
```

Comment:
Nit wording: `... is set to true, [DeltaTextInputClient] must be implemented instead of this class.`